src/pages/javascript/es6/arrow-functions/index.md

```diff
@@ -37,4 +37,12 @@ let NewOneWithParameters = (a, b) => {
 NewOneWithParameters(10, 20);
 ```
 
+Parentheses are optional when there's only one parameter name:
+
+```
```

Comment:
Hi @fangcha 

Thank you for opening this PR. Can you please edit this to use JS Code Highlighting? You can read the [Article Style Guide](https://github.com/freeCodeCamp/guide/blob/master/CONTRIBUTING.md#code-blocks) on how to change the markdown to support it.

Can you also add it to `line 9` and `line 33`? I noticed that they aren't set up to support the correct syntax highlighting.

Thank you

Reply:
Hi @Asjas,

Edited as requested! Thanks very much.